contracts/evm/hardhat.config.ts

```diff
@@ -174,6 +202,7 @@ const config: HardhatUserConfig = {
     }
   },
   etherscan: {
+    // enabled: false, // uncomment this for Sophon verification
```

Comment:
any way we could just add something to the sophonMainnet config so editing this isn't necessary? or like not do the standard verification if verifyURL is set?

Reply:
doesnt appear to work right now, this is the workaround they gave me. Hopefully the tooling improves and we can remove this